tools/maxquant/maxquant.xml

```diff
@@ -310,17 +310,29 @@ This tool is a wrapper for MaxQuant v@VERSION@. The current version of the wrapp
 - Thermo raw file or mzXML file
     - The datatype has to be 'thermo.raw' or 'mzXML'. Make sure to specify the correct datatype either during upload to Galaxy or afterwards (edit attributes --> datatypes)
 - Optional files:
-    - Tabular file with experimental design template: 
-        - Currently four columns are needed: Name, Fraction, Experiment and PTM. The headers must have this exact naming. Name and Fraction are abitrary strings, Experiment is an integer, PTM is either True or False.
+    - Tabular file with experimental design template:
+        - Currently four columns are needed: Name, Fraction, Experiment and PTM. The headers must have this exact naming. Name and Experiment are abitrary strings, Fraction is an integer or emtpy, PTM is either 'True', 'False' or empty. Consider you uploaded files named File1.mzxml, ..., File5.mzxml. This is a (syntactically) correct experimental design template:
 
                 ::
 
                     Name     Fraction    Experiment   PTM
                     File1        1          E1       False
                     File2        2          E1       False
+                    ghost       234        none
                     File3        3          E1       False
-                    ...
-                    ...
+                    File4                   E2       true
+                    File5                   E1
+
+        - This is the counter-example:
```

Comment:
counter example means this will not work?

Reply:
yes, one error per line